docs/bdd.md

```diff
@@ -175,6 +175,24 @@ Then('I should see that total number of products is {int}', (num) => {
 Then('my order amount is ${int}', (sum) => { // eslint-disable-line
   I.see('Total: ' + sum);
 });
+
```

Comment:
Could you add some paragraphs of text about what are custom parameter types and when you suggest using them.
Sure, you can use the Color as an example but please add a new section in docs about it. 

I recommend adding it as a section to Advanced Gherkin 

https://codecept.io/bdd/#advanced-gherkin

Reply:
@DavertMik I added section and give more advanced example.